mmv1/templates/terraform/examples/looker_instance_enterprise_full.tf.erb

```diff
@@ -0,0 +1,76 @@
+resource "google_looker_instance" "<%= ctx[:primary_resource_id] %>" {
+  name               = "<%= ctx[:vars]["instance_name"] %>"
+  platform_edition   = "LOOKER_CORE_ENTERPRISE_ANNUAL"
+  region             = "us-central1"
+  private_ip_enabled = true
+  public_ip_enabled  = false
+  reserved_range     = "${google_compute_global_address.looker_range.name}"
+  consumer_network   = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.looker_network.name}"
```

Comment:
Does this accept a project id, or just a number? Per https://google.aip.dev/cloud/2510#google-apis it should accept both (and return the format that was sent), and project ids are generally preferred as we can just reference https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network#id instead of needing to do string interpolation here.

Reply:
@rileykarson Done!